internal/testplanet/uplink.go

```diff
@@ -298,10 +298,10 @@ func (uplink *Uplink) GetConfig(satellite *satellite.Peer) uplink.Config {
 	config.Client.SatelliteAddr = satellite.Addr()
 	config.Client.APIKey = uplink.APIKey[satellite.ID()]
 
-	config.RS.MinThreshold = 1 * uplink.StorageNodeCount / 5
-	config.RS.RepairThreshold = 2 * uplink.StorageNodeCount / 5
-	config.RS.SuccessThreshold = 3 * uplink.StorageNodeCount / 5
-	config.RS.MaxThreshold = 4 * uplink.StorageNodeCount / 5
+	config.RS.MinThreshold = atLeastOne(1 * uplink.StorageNodeCount / 5)
```

Comment:
I guess that `1 * ..` is to make clear something, isn't it?

Reply:
If we move the multiplication to the end it should become more obvious `uplink.StorageNodeCount * 1 / 5`

Reply:
from my point of view is the same, I mostly was asking specifically for the multiplication by 1

Reply:
Comments added. Let me know if it requires more clarification.